test/TLX/attach-metadata_implicit_tlx.mlir

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
nit: can we merge the three test files into one as they are testing the same pass?

Reply:
you’d need  -split-input-file  in the triton-opt command line, something like https://github.com/triton-lang/triton/blob/8dec0edf6a0e9cac424e3fce9d0c61e42ab248c9/test/Hopper/WarpSpecialization/ws_data_partition.mlir#L1